JobConfig/primary/IPAStopParticle.fcl

```diff
@@ -8,6 +8,7 @@
 physics.filters.TrackerRadialGenFilter: {
   @table::GenFilter
   isNull: false
+  filterMaxR : true
```

Comment:
For future reference, to make a cut optional you can use art::OptionalAtom<> to encode both the value and whether the cut should be applied. That avoids having a separate bool and the possibility of a self-inconsistent config (value set but boolean not).

Reply:
Interesting, I didn't realize this! I can make a quick PR to fix this, otherwise I'll make sure to do this next time.

Reply:
If you can patch this quickly in Offline that would be best.